integration-tests/dataformats-json/src/main/java/org/apache/camel/quarkus/component/jackson/JsonDataformatsRoute.java

```diff
@@ -16,47 +16,93 @@
  */
 package org.apache.camel.quarkus.component.jackson;
 
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.google.gson.ExclusionStrategy;
+import com.google.gson.FieldAttributes;
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.reflect.TypeToken;
 import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.gson.GsonDataFormat;
 import org.apache.camel.component.jackson.JacksonDataFormat;
 import org.apache.camel.model.dataformat.JsonLibrary;
 import org.apache.camel.quarkus.component.jackson.model.DummyObject;
+import org.apache.camel.quarkus.component.jackson.model.ExcludeField;
 import org.apache.camel.quarkus.component.jackson.model.PojoA;
 import org.apache.camel.quarkus.component.jackson.model.PojoB;
+import org.apache.camel.spi.DataFormat;
 
 public class JsonDataformatsRoute extends RouteBuilder {
 
     @Override
     public void configure() {
-        JacksonDataFormat format = new JacksonDataFormat(DummyObject.class);
-        format.useList();
+        JacksonDataFormat jacksonDummyObjectDataFormat = new JacksonDataFormat(DummyObject.class);
```

Comment:
Why do these data format variables have the `Dummy` infix in their name? I do not see anything "dummy" in how they are used.

Reply:
The idea is that it's the dataformat serializing dummy object. That said, I would not veto a renaming of DummyObject too.

Reply:
Makes sense. Thanks for the explanation :)